src/pke/include/schemerns/rns-cryptoparameters.h

```diff
@@ -581,13 +581,7 @@ class CryptoParametersRNS : public CryptoParametersRLWE<DCRTPoly> {
   double GetScalingFactorReal(uint32_t l = 0) const {
     if (m_rsTechnique == FLEXIBLEAUTO) {
       if (l >= m_scalingFactorsReal.size()) {
-        OPENFHE_THROW(math_error,
```

Comment:
What do you think the behavior should be when this is the case? I put in m_approxSF as a placeholder because I didn't want it to throw an exception when l >= m_scalingFactorsReal.size().

Basically, this occurs when GetScalingFactorReal is called in rns-leveledshe.cpp by BGV-RNS. GetScalingFactorReal is only really used in CKKS-RNS though.